cdk/tooling_environment/stacks/grafana/lambda_functions_dashboard.template.json

```diff
@@ -15,10 +15,111 @@
+            "steps": [
+              {
+                "color": "green",
+                "value": null
+              }
+            ]
+          }
+        },
+        "overrides": [
+          {
+            "matcher": {
+              "id": "byName",
+              "options": "Total Duration, sec"
+            },
+            "properties": [
+              {
+                "id": "decimals",
+                "value": 3
+              }
+            ]
+          },
+          {
+            "matcher": {
+              "id": "byName",
+              "options": "Total Billed Duration, sec"
+            },
+            "properties": [
+              {
+                "id": "decimals",
+                "value": 3
+              }
+            ]
+          },
+          {
+            "matcher": {
+              "id": "byName",
+              "options": "Total GB-seconds"
+            },
+            "properties": [
+              {
+                "id": "decimals",
+                "value": 3
+              }
+            ]
+          }
+        ]
+      },
+      "gridPos": {
+        "h": 3,
+        "w": 12,
+        "x": 0,
+        "y": 0
+      },
+      "id": 2,
+      "options": {
+        "colorMode": "value",
+        "graphMode": "area",
+        "justifyMode": "auto",
+        "orientation": "vertical",
+        "reduceOptions": {
+          "calcs": [
+            "lastNotNull"
+          ],
+          "fields": "",
+          "values": false
+        },
+        "text": {
+          "titleSize": 18,
+          "valueSize": 40
+        },
+        "textMode": "value_and_name",
+        "wideLayout": false
+      },
+      "pluginVersion": "10.2.2",
+      "targets": [
+        {
+          "database": "<<DATABASE_NAME>>",
+          "datasource": "Amazon-Timestream",
+          "measure": "execution",
+          "rawQuery": "SELECT sum(execution)as \"Total executions\", sum(succeeded) as \"Succeeded executions\", sum(failed) as \"Failed executions\", sum(retry_attempts) as \"Retry attempts\"\r\nFROM(\r\nSELECT 1 as execution\r\n ,(sum(invocation) over (partition by lambda_function_request_id)) -1 as retry_attempts\r\n , min(failed) over (partition by lambda_function_request_id) as failed\r\n , max(succeeded) over (partition by lambda_function_request_id) as succeeded\r\n, sum(duration_ms) over (partition by lambda_function_request_id) as total_duration_ms\r\n, sum(billed_duration_ms) over (partition by lambda_function_request_id) as total_billed_duration_ms\r\n, sum(memory_size_mb) over (partition by lambda_function_request_id) as total_memory_size_mb\r\n, sum(GB_seconds) over (partition by lambda_function_request_id) as  total_GB_seconds\r\n, max(max_memory_used_mb) over (partition by lambda_function_request_id) as max_memory_used_mb FROM $__database.$__table where $__timeFilter and resource_name in (${resource_name:singlequote}) and monitored_environment in (${monitored_env:singlequote})) t\r\n",
+          "refId": "A",
```

Comment:
query for readability:
![image](https://github.com/user-attachments/assets/f271ff25-a06e-4ec2-bb90-8eba730873aa)